pkg/netlight/network.go

```diff
@@ -147,17 +204,17 @@ func (n *networker) AttachZDB(id string) (string, error) {
 
 // GetSubnet of a local network resource identified by the network ID, ipv4 and ipv6
 // subnet respectively
-func (n *networker) GetSubnet(networkID pkg.NetID) (net.IPNet, error) {
-	localNR, err := n.networkOf(networkID)
+func (n *networker) GetSubnet(name string) (net.IPNet, error) {
```

Comment:
netid was created as defined type to be easy in the future if we decided to change it to something else
also we still use it in other places as netid not string which brings inconsistency to the code base